### PR TITLE
[WIP] first version of the single script with example 1B for B1F4

### PR DESCRIPTION
**Pull request by zonca**
workflow:

* create a folder for the molecule type, e.g. `B1F4`
* create a configuration file `config.ini` with all parameters (see example below)
* run `python run.py config.ini`, this does all the steps

still rough, but a lot to discuss, better to talk, @agoetz can you invite who might be interested and then we can have a call via Zoom next Wed?

### CODE REVIEW

File: B1F4/config.ini [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+excluded_pairs_12 = [[0,1],[0,2],[0,3],[0,4]]
+excluded_pairs_13 = [[1,2],[1,3],[1,4],[2,3],[2,4],[3,4]]
+excluded_pairs_14 = []
+#
+#Obtain charges (in the order of input), pols and polfacs
+charges = [0.326705,-0.331688,-0.331688,-0.331688,-0.331688]
+polarizabilities = [3.8767477858,0.5333436504,0.5333436504,0.5333436504,0.5333436504]
+polarizability_fractions = [3.8767477858,0.5333436504,0.5333436504,0.5333436504,0.5333436504]
+#
+k_min = 0.0
```
Comment (zonca) [inline B1F4/config.ini]: need better names for this

File: B1F4/config.ini [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+C6 = [310.915,216.702,172.445]
+d6 = [3.20676,3.38985,3.74833]
+#
+## Define kind of variables 
+## coul is e^(-k(d-d0)/d)
+## exp  is e^(-k(d-d0))
+var = exp
+#
+## Define Energy Range for the fitting
+energy_range = 50.0
```
Comment (zonca) [inline B1F4/config.ini]: does this mean from zero to 50? Unit?

File: B1F4/config.ini (hunk 1 above)
Comment (agoetz) [reply B1F4/config.ini]: Currently the variable names are consistent with README.md files, e.g. fitting/1B/README.md. I only would change d_min and d_max into d0_min and d0_max to match the equations that define the variables in the comment.

File: B1F4/config.ini (hunk 2 above)
Comment (agoetz) [reply B1F4/config.ini]: This is the range of energies, starting at the minimum energy, that are favorably weighted in the fit. It is \Delta E in Equation (10) of doi: 10.1021/ct400863t

Comment (agoetz) [reply B1F4/config.ini]: Unit is the unit of energies in the training set. The user has to know. In general we work with kcal/mol.

**Comment by agoetz** [comment]
Looks good, I will merge although I did not test everything since I don't have maple installed. I guess this is also why we don't have travis ci for the python workflow?

**Pull request merged**